Packs/Slack/Integrations/SlackV3/SlackV3_test.py

```diff
@@ -4092,3 +4110,151 @@ async def test_fetch_channels_paginated_async(mocker):
 
     # Assert
     assert returned_time.replace(microsecond=0) >= (last_update + timedelta(seconds=8)).replace(microsecond=0)
+
+
```

Comment:
general comment, we need to consider a test that checks that user list is merged correctly.(after the fix of the `.extend(updated_users)`)